lib/internal/bootstrap_node.js

```diff
@@ -78,11 +78,13 @@
         NativeModule.require('_third_party_main');
       });
 
-    } else if (process.argv[1] === 'debug') {
-      // Start the debugger agent
-      NativeModule.require('_debugger').start();
+    } else if (process.argv[1] === 'inspect' || process.argv[1] === 'debug') {
+      if (process.argv[1] === 'debug') {
+        process.emitWarning(
+          '`node debug` is deprecated. Please use `node inspect` instead.',
+          'DeprecationWarning', 'DEP0062');
```

Comment:
Technically DEP0062 is for `node --inspect` (activate the websocket server) and `node debug` (use the builtin CLI debugger) should get another one.